packages/shared/routes/login/Login.svelte

```diff
@@ -49,6 +50,7 @@
         if (prevState) {
             state = prevState
         } else {
+            state = LoginState.Init
```

Comment:
why is this line needed here?

Reply:
If you start the app it does not have a previous state, so when the EnterPin screen is active I need a way to return to the main screen with the profile list